pkg/k8shandler/desirednodestate.go

```diff
@@ -239,6 +272,33 @@ func (cfg *desiredNodeState) CreateOrUpdateNode(owner metav1.OwnerReference) err
 			return fmt.Errorf("Error: could not update status for Elasticsearch %v after %v retries: %v", cfg.DeployName, nretries, retryErr)
 		}
 		logrus.Debugf("Updated Elasticsearch %v after %v retries", cfg.DeployName, nretries)
+
+		// set it back to being paused
+		cfg.Paused = true
+		if getErr := node.query(); getErr != nil {
```

Comment:
same comments here as above about `node.query()` and the retry loop and the `Infof` message